packages/components/src/tooltip/test/index.js

```diff
@@ -1,19 +1,24 @@
 /**
  * External dependencies
  */
-import { render, screen, act, waitFor } from '@testing-library/react';
+import { render, screen, waitFor } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 
 /**
  * Internal dependencies
  */
 import Tooltip from '../';
+import Button from '../../button';
+
 /**
  * WordPress dependencies
  */
 import { TOOLTIP_DELAY } from '../index.js';
 
-jest.useFakeTimers();
+const props = {
```

Comment:
I think we could add the anchor as a prop to repeat a bit less in each test which was my intention with adding the tooltip text